src/hooks/useScreenRecorder.ts

```
@@ -1131,20 +1131,45 @@ export function useScreenRecorder(): UseScreenRecorderReturn {
+						return await navigator.mediaDevices.getUserMedia({
+							audio: false, //intentional
+							video: {
+								mandatory: {
+									chromeMediaSource: "desktop",
+									chromeMediaSourceId: source.id,
+									maxWidth: TARGET_WIDTH,
+									maxHeight: TARGET_HEIGHT,
+									maxFrameRate: TARGET_FRAME_RATE,
+								},
+							},
+						} as any);
```

Biome (2.4.13)

[error] Unexpected any. Specify a different type.

(lint/suspicious/noExplicitAny)